backends/events/sdl/sdl-events.cpp

```diff
@@ -81,7 +81,8 @@ SdlEventSource::SdlEventSource()
 	if (joystick_num >= 0) {
 		// Initialize SDL joystick subsystem
 		if (SDL_InitSubSystem(SDL_INIT_JOYSTICK) == -1) {
-			error("Could not initialize SDL: %s", SDL_GetError());
+			warning("Could not initialize SDL joystick: %s", SDL_GetError());
```

Comment:
It might be a good idea to allow `SDL_InitSubSystem(SDL_INIT_GAMECONTROLLER)` to fail as well.

Reply:
Done